python/ray/serve/tests/test_pipeline_driver.py

```diff
@@ -15,6 +15,7 @@ def my_resolver(a: int):
     return a
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Fails on windows.")
```

Comment:
This whole feature is reverted in PR #23358. It was added quite recently. For now skip the failing test